### PR TITLE
Update build scripts

### PR DESCRIPTION
**Pull request by mnaamani**
Some minor polishing of README files and correcting ref to rust nightly version.

Pulled from sumer branch cleaner build scripts and updated READMEs.

### CODE REVIEW

File: .github/workflows/joystream-types.yml [hunk 1]
```diff
@@ -40,6 +41,9 @@ jobs:
     - name: checks
       run: |
         yarn install --frozen-lockfile --network-timeout 120000
+        yarn workspace @joystream/types build
+        yarn workspace @joystream/cd-schemas generate:all
```
Comment (Lezek123) [inline .github/workflows/joystream-types.yml]: We don't need to build cd-schemas here

File: .github/workflows/network-tests.yml [hunk 2]
```diff
@@ -17,6 +17,10 @@ jobs:
     - name: checks
       run: |
         yarn install --frozen-lockfile
+        yarn workspace @joystream/types build
+        yarn workspace @joystream/cd-schemas generate:all
+        yarn workspace @joystream/cd-schemas build
+        yarn workspace @joystream/cli build
```
Comment (Lezek123) [inline .github/workflows/network-tests.yml]: Is there a reason for building the CLI here?

File: .github/workflows/network-tests.yml [hunk 3]
```diff
@@ -34,4 +38,8 @@ jobs:
     - name: checks
       run: |
         yarn install --frozen-lockfile --network-timeout 120000
+        yarn workspace @joystream/types build
+        yarn workspace @joystream/cd-schemas generate:all
+        yarn workspace @joystream/cd-schemas build
+        yarn workspace @joystream/cli build
```
Comment (Lezek123) [inline .github/workflows/network-tests.yml]: Is there a reason for building the CLI here?

File: .github/workflows/run-network-tests.yml [hunk 4]
```diff
@@ -171,9 +179,13 @@ jobs:
...
-        run: yarn install --frozen-lockfile
-      - name: Ensure query-node builds
-        run: yarn workspace query-node-root build
+        run: |
+          yarn install --frozen-lockfile
+          yarn workspace @joystream/types build
+          yarn workspace @joystream/cd-schemas generate:all
+          yarn workspace @joystream/cd-schemas build
+          yarn workspace query-node-root build
+          yarn workspace @joystream/cli build
```
Comment (Lezek123) [inline .github/workflows/run-network-tests.yml]: Is there a reason for building the CLI here?

File: .github/workflows/network-tests.yml (hunk 2 above)
Comment (mnaamani) [reply .github/workflows/network-tests.yml]: no, will drop

File: .github/workflows/network-tests.yml (hunk 3 above)
Comment (mnaamani) [reply .github/workflows/network-tests.yml]: no will drop

File: .github/workflows/run-network-tests.yml (hunk 4 above)
Comment (mnaamani) [reply .github/workflows/run-network-tests.yml]: I don't think query-node tests currently use the cli so will remove

File: .github/workflows/joystream-types.yml (hunk 1 above)
Comment (mnaamani) [reply .github/workflows/joystream-types.yml]: indeed

File: .github/workflows/run-network-tests.yml (hunk 4 above)
Comment (mnaamani) [reply .github/workflows/run-network-tests.yml]: done in https://github.com/Joystream/joystream/pull/2302/commits/d9d6762b817b9d1e8c4867bbaa7379499dfcf9c6

File: .github/workflows/network-tests.yml (hunk 3 above)
Comment (mnaamani) [reply .github/workflows/network-tests.yml]: done in https://github.com/Joystream/joystream/pull/2302/commits/d9d6762b817b9d1e8c4867bbaa7379499dfcf9c6

File: .github/workflows/network-tests.yml (hunk 2 above)
Comment (mnaamani) [reply .github/workflows/network-tests.yml]: done in https://github.com/Joystream/joystream/pull/2302/commits/d9d6762b817b9d1e8c4867bbaa7379499dfcf9c6

File: .github/workflows/joystream-types.yml (hunk 1 above)
Comment (mnaamani) [reply .github/workflows/joystream-types.yml]: done in https://github.com/Joystream/joystream/pull/2302/commits/d9d6762b817b9d1e8c4867bbaa7379499dfcf9c6

**Review by Lezek123 — APPROVED** [review]

**Pull request merged**